MaxText/configs/base.yml

```diff
@@ -109,3 +109,9 @@ prompt: "I love to "
 enable_profiler: False
 enable_checkpointing: True
 
+grad_cut_factor: 1.2 # Gradient capped by mean of previous gradients times this factor.
```

Comment:
`grad_cut` is a bad name unfortunately. My best is `rolling_gradient_clipping`.

Reply:
So we'd call the various quantities lower in the code `rolling_gradient_clipping_window_len`, `rolling_gradient_clipping_start_step`, `rolling_gradient_clipping_scale_factor`, `rolling_gradient_clipping_current_threshold`, etc.